platforms/linux/qlcplus.spec

```diff
@@ -39,6 +39,7 @@ of other lighting control commercial softwares.
 #############################################################################
 
 %build
+rm .qmake.stash
```

Comment:
To be honest I never had issues with qmake.stash
What do you mean with "Keeping it prevents building on other platforms"?
When you build QLC+ you are targeting a single platform and you are supposedly starting from a clean checkout of the repository, which doesn't include qmake.stash.
I would revert this commit.

Reply:
I understand that the current main case for create-rpm.sh is to build locally for the current platform. But currently, it still works by creating a "clean" tar and build the rpm "from scratch" in the lightweight rpmbuild "sandbox".
I think it will save a lot of trouble to do that cleanly.
For example, I tried to verify that the same source rpm could build in a real sandbox on "all" hardware platforms. It failed as https://koji.fedoraproject.org/koji/taskinfo?taskID=60937238 when the source rpm contained an x86_64 .qmake.stash . build.log shows for example
```
g++ -pipe -Werror -Wno-unused-local-typedefs -g -fPIC -Wall -Wextra -dM -E -o moc_predefs.h /../lib/qt5/mkspecs/features/data/dummy.cpp
/../lib/qt5/bin/moc -DQT_GUI_LIB -DQT_CORE_LIB --include /builddir/build/BUILD/qlcplus-4.12.4/hotplugmonitor/src/moc_predefs.h -I/../lib/qt5/mkspecs/linux-g++ -I/builddir/build/BUILD/qlcplus-4.12.4/hotplugmonitor/src -I/usr/include/qt5 -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I/usr/include/c++/10 -I/usr/include/c++/10/x86_64-redhat-linux -I/usr/include/c++/10/backward -I/usr/lib/gcc/x86_64-redhat-linux/10/include -I/usr/local/include -I/usr/include hpmprivate-udev.h -o moc_hpmprivate-udev.cpp
usr/include/c++/10/bits/stl_relops.:67: Parse error at "std"
make[2]: *** [Makefile:451: moc_hpmprivate-udev.cpp] Error 1
```

Reply:
I haven't been able to reproduce or understand how this could happen. I will leave out this change.